wepool-app/src/screens/RideDisplay.tsx

```diff
@@ -1,30 +1,57 @@
 import { useState, useEffect } from 'react'
 import {HeaderBar} from "../components/HeaderBar";
-import { StyleSheet, View, ScrollView } from 'react-native'
+import { StyleSheet, View, ScrollView, Text } from 'react-native'
 import { RootTabScreenProps } from '../navigation/types';
 import {BackButton} from '../components/BackButton'
 import {Header} from "../components/Header";
 import {SearchBar} from '../components/SearchBar'
 import {RideCard} from '../components/RideCard';
 import _testUsers from '../TestDummyUsers.json';
+import { useQuery } from '@apollo/client';
+
+    // queries
+    import GetOpenRides from '../queries/GET/RideQueries'
 
 export const RideDisplay = ({navigation}: RootTabScreenProps<'RideDisplay'>) => {
 
     /**
-     * Getting dummy users info for testing purposes. 
      * TODO: 
-     *  - Integrate with backend
+     *  - Query cant access to 'status' variable.
```

Comment:
I thought this was fixed? Do we have the query only on sandbox?

Reply:
Yes, its fixed